### PR TITLE
/community fixes

### PR DESCRIPTION
**Pull request by caldav**
Typo, layout and wording fixes

## Done

* Missing semicolon
* s/snappy/snapcraft
* Boxes layout

## Screenshots

screenshot from 2017-04-18 10-57-13

### CODE REVIEW

File: community/index.html [hunk 1]
```diff
@@ -22,30 +22,30 @@ <h1 class="hero-row__heading">Community and support</h1>
             <div class="equal-height--vertical-divider">
 
                 <div class="six-col box">
-                    <h3><a href="https://forum.snapcraft.io">Talk about Snapcraft on the forum&nbsp;&rsaquo</a></h3>
-                    <p>Got ideas for how Snapcraft could be improved? Want to be an active part of the community and get involved? 
-                    Head over to the official <a class="external" href="https://forum.snapcraft.io">Snapcraft forum</a> 
+                    <h3><a href="https://forum.snapcraft.io">Talk about Snapcraft on the forum &rsaquo;</a></h3>
```
Comment (anthonydillon) [inline community/index.html]: It's good to leave the `&nbsp;` here to stop the angle bracket from wrapping on its own. Good catch with the missing trailing `;`

File: community/index.html [hunk 2]
```diff
@@ -22,30 +22,30 @@ <h1 class="hero-row__heading">Community and support</h1>
...
                     <p>If you’re stuck, the chances are that someone else has encountered
                     the same problem and they can help. Take a look at the
                     <a class="external" href="http://stackoverflow.com/questions/tagged/snapcraft">"snapcraft" tag</a>
                     on Stack Overflow.</p>
                 </div>
 
-                <div class="six-col last-col box">
-                    <h3><a href="https://rocket.ubuntu.com/channel/snapcraft">Or chat in real time ›</a></h3>
+                <div class="six-col box">
+                    <h3><a href="https://rocket.ubuntu.com/channel/snapcraft">Chat in real time &rsaquo;</a></h3>
```
Comment (anthonydillon) [inline community/index.html]: Please add a `&nbsp;` before the angle bracket.

File: community/index.html [hunk 3]
```diff
@@ -22,30 +22,30 @@ <h1 class="hero-row__heading">Community and support</h1>
...
                     <p>Share your projects and ask other developers for support. This
                     high-bandwidth chat room is a great place to find <a class="external"
                     href="https://rocket.ubuntu.com/channel/snapcraft">a quick answer</a>
                     to a single question.</p>
                 </div>
 
                 <div class="six-col last-col box">
-                    <h3><a href="https://lists.snapcraft.io/mailman/listinfo/snapcraft">Join the mailing list ›</a></h3>
+                    <h3><a href="https://lists.snapcraft.io/mailman/listinfo/snapcraft">Join the mailing list &rsaquo;</a></h3>
```
Comment (anthonydillon) [inline community/index.html]: `&nbsp;` please.

**Comment by caldav** [comment]
Suggestions&nbsp;applied

**Pull request merged**